contrib/emicklei/go-restful/restful_test.go

```diff
@@ -0,0 +1,132 @@
+package restful
+
+import (
+	"errors"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/emicklei/go-restful"
+	"github.com/stretchr/testify/assert"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/ext"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/mocktracer"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
+)
+
+func TestChildSpan(t *testing.T) {
```

Comment:
You can remove this test. You are already testing this behavior in the following one, right?

Reply:
yes, you are correct, will remove this too